features/src/main/java/com/mupper/features/bus/GetTravelingBus.kt

```diff
@@ -0,0 +1,19 @@
+package com.mupper.features.bus
+
+import com.mupper.data.source.firestore.BusRemoteDataSource
+import com.mupper.data.source.room.BusLocalDataSource
+import com.mupper.domain.relations.BusWithTravelers
+
+class GetTravelingBus(
```

Comment:
Is this a repository? In that case, wouldn't it be convenient to use the suffix "Repository"?

Reply:
That is a feature, in others words, is a use case, so, I think in practice is the same think a Repository and a Use case, it depends on your methodology.

What I wanted to implement is a Use Case